packages/webcert/src/feature/certificate/CertificateHeader/CertificateHeader.tsx

```diff
@@ -16,7 +16,7 @@ import styled from 'styled-components/macro'
 import { Divider } from '@frontend/common'
 import CreateCertificateFromCandidateModal from '../Modals/CreateCertificateFromCandidateModal'
 import { resourceLinksAreEqual, ResourceLinkType } from '@frontend/common/src'
-import { getQuestions } from '../../../store/question/questionSelectors'
+import { getQuestions, isQuestionFunctionDisabled } from '../../../store/question/questionSelectors'
```

Comment:
Should this really be isQuestionFunctionDisabled? The CertificateHeader should enable/disable based on the certificate.